modules/ROOT/pages/amazon/amazon-sqs-connector-examples.adoc

```diff
@@ -0,0 +1,291 @@
++
+. Select the *Configuration XML* tab to view the corresponding XML:
++
+[source,dataweave,linenums]
+----
+<sqs:config name="Amazon_SQS_Configuration"
+  doc:name="Amazon SQS Configuration"
+	defaultQueueUrl="${sqs.queueUrl}">
+	<sqs:basic-connection accessKey="${sqs.accessKey}"
+	secretKey="${sqs.secretKey}"
+	region="us-east-1"/>
+</sqs:config>
+----
++
+. Configure the Following fields on the properties window:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Name for the connector operation.
+|Connector Configuration |Global configuration you just created
+|Message |`payload`
+|Queue url |Amazon SQS queue URL. If provided, the value of this field takes precedence over the value of the *Default Global Queue URL* field on the *Global Configuration Elements* window.
+|===
++
+For example:
++
+image::amazon/amazon-sqs-send-message.png[send-message]
+
+=== Add a Logger Component to Display the Response in the Mule Console
+
+. In the Mule Palette, search for *Logger*.
+. Drag the component onto the canvas, to the right of the *Send Message* component.
+. Configure the following fields:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Name for the logger, such as `Log Response`
+|Message |`+++Sent Message: `#[payload]`+++`
+|Level |INFO (Default)
+|===
++
+For example:
++
+image::amazon/amazon-sqs-logger.png[logger]
+
+=== Obtain the Number of Messages in the Queue
+
+. In the Mule Palette, search for *Amazon SQS*.
+. Select the *Get approximate number of messages* operation and drag it onto the canvas, to the right of the *Logger* component.
++
+. Configure the following fields on the properties window:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Queue url |Amazon SQS queue URL
+|===
++
+For example:
++
+image::amazon/amazon-sqs-get-message-count.png[get-message-count]
+
+=== Add a Logger to Display the Number in the Mule Console
+
+. In the Mule Palette, search for *Logger*.
+. Drag the component onto the canvas, to the right of the *Get approximate number of messages* component.
+. Configure the following fields:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Enter a name for the logger, such as `Log Count`
+|Message |`+++Approx. messages in queue: `#[payload]`+++`
+|Level |INFO (Default)
+|===
++
+For example:
++
+image::amazon/amazon-sqs-logger2.png[log-message-count]
+
+== Create a Flow to Receive Messages
+
+Create another flow to receive messages and log them before deleting them from the queue.
+
+. In the Mule Palette, search for *SQS* and select the *Receive messages* operation.
++
+image::amazon/amazon-sqs-select-receive.png[select-receive-messages]
++
+. Drag the *Receive messages* operation onto the canvas.
+. Configure the following fields in the properties window:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Name for the connector operation.
+|Connector Configuration |Global configuration you created previously for Amazon SQS Connector
```

Comment:
```suggestion
|Connector Configuration |Global configuration you created previously for Amazon SQS Connector.
```

Reply:
I am not sure about the period here. I will leave this one for Kim.